core/jobs.py

```diff
@@ -1141,11 +1150,13 @@ def _handle_incoming_event(
 
     @classmethod
     def _get_active_realtime_index(cls):
-        """Returns the registered realtime id of this class, creating one if it
-        doesn't exist.
+        """Returns the currently active realtime layer index for this class.
+
+        Additionally takes care to register the class into the model when it
```

Comment:
Additionally, takes care to register the relevant ContinuousComputation model, if it does not already exist.